bugbear.py

```diff
@@ -168,6 +181,7 @@ class BugBearVisitor(ast.NodeVisitor):
     node_window = attr.ib(default=attr.Factory(list))
     errors = attr.ib(default=attr.Factory(list))
     futures = attr.ib(default=attr.Factory(set))
+    context = attr.ib(default=attr.Factory(list))
```

Comment:
Is this all that is needed? How did this ever work if so :|
```suggestion
    contexts = attr.ib(default=attr.Factory(list))
```